src/sentry/incidents/logic.py

```diff
@@ -988,67 +987,61 @@ def trigger_incident_triggers(incident):
                         incident_id=incident.id,
                         project_id=project.id,
                         method="resolve",
+                        new_status=IncidentStatus.CLOSED.value,
                     ).delay
                 )
 
 
-def sort_by_priority_list(incident_triggers):
+def sort_by_priority_list(triggers: list[AlertRuleTrigger]) -> list[AlertRuleTrigger]:
     priority_dict = {
```

Comment:
Got rid of the prioritization within trigger types. Currently, you can't have multiple triggers of the same type afaik. Even if this is or becomes possible, it doesn't matter because all of the unique actions will be sent now.